docs/ops.html

```diff
@@ -586,7 +582,7 @@
   socket.request.max.bytes=104857600
   socket.receive.buffer.bytes=1048576
   socket.send.buffer.bytes=1048576
-  queued.max.requests=16
+  queued.max.requests=500
```

Comment:
Properly increasing this config might be a help since multiple ReplicaFetcherThread instance will be created, where multiple underlying network clients will invoke the blocking `select` simultaneously, so it might improve the throughput for replicating somewhat. Does it make sense?